Makefile

```
@@ -104,6 +104,9 @@ assets-clean:
 assets: repo/assets
 	mkdir -p $(testdir)/assets
 	cp -r -t $(testdir)/assets repo/assets/data/*
+	mkdir -p models
+	make model
+	cp -r  models/ $(testdir)/assets/dfki-testdata/data/
```

If your concern is with disk I/O bandwidth, how about:
```suggestion
	cp -r --reflink=auto  models/ $(testdir)/assets/dfki-testdata/data/
```

This uses CoW semantics, falling back to phyisical copies if the FS/OS does not support it.

---

I cannot make suggestions outside the context, but if you add the models as prerequisites for the CLI tests individually, you can safely remove the `make model` line here.

---

Did not know about `--reflink=auto`, thanks.  This should be the default IMHO (I've already adapted my shell aliases 😄).